.travis.yml

```diff
@@ -5,6 +5,7 @@ before_install:
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 install:
+  - npm install bitcore
```

Comment:
I don't think this will be necessary.  It's only in the case that it's required in a package.json, and a peer, that it would expect to already be available as a peer dependency.